easybuild/easyconfigs/t/TensorFlow/TensorFlow-2.15.1-foss-2023b.eb

```diff
@@ -69,7 +68,7 @@ exts_list = [
         'checksums': ['aedf21eec7fb2dc91150fc91a1ce12bc44dbb72278a08b58e79ff87c9e28f153'],
     }),
     ('Werkzeug', '3.0.2', {
-        'source_tmpl': SOURCELOWER_TAR_GZ,
+        'source_tmpl': '%(namelower)s-%(version)s.tar.gz',
```

Comment:
Why this change?

Reply:
Honestly the change was not intentional and also not manual. It appeared when I did: "eb --inject-checksums=sha256 --force Tensorflow-2.15.1-foss-2023b.eb" to include the two new patches - probably due to formatting by EasyBuild/Python?

Reply:
I opened an issue for that: https://github.com/easybuilders/easybuild-framework/issues/4695